shopify bot helpers

cshape opened the pull request:
example bot and FaaS function for shopify bots

JasonSowers commented on bot-samples/shopify-bot-helpers/README.md:
```diff
@@ -0,0 +1,53 @@
+# Instructions
+
+1. Create a custom app for your Shopify store and get the authorization token or URL
+2. Update the 1Shopify2ContextService.js` FaaS function with your Shopify/account information, deploy it, and run it. Ensure that you whitelist `*.context.liveperson.net` and `*.myshopify.com`
```
I think you should add where to add these to the whitelist

cshape replied:
added

JasonSowers commented on bot-samples/shopify-bot-helpers/README.md:
```diff
@@ -0,0 +1,53 @@
+5. In the bot's global function, update the `storeURL` variable to the URL for your store (eg. `https://mystorename.myshopify.com`)
+5. If you have access to Houston, whitelist `https://cdn.shopify.com`. Otherwise, contact your LivePerson representative.
+6. Test the bot in the Previewer or in a Web Engagement. Try adding multiple items and then checking out.
+
```
Since there is a lot of comments that say add your _____ here. Maybe add a step that says read through the code and add the values the comments ask for.

cshape replied:
added

JasonSowers commented on bot-samples/shopify-bot-helpers/README.md:
```diff
@@ -0,0 +1,53 @@
+5. In the bot's global function, update the `storeURL` variable to the URL for your store (eg. `https://mystorename.myshopify.com`)
+5. If you have access to Houston, whitelist `https://cdn.shopify.com`. Otherwise, contact your LivePerson representative.
+6. Test the bot in the Previewer or in a Web Engagement. Try adding multiple items and then checking out.
+
```
There is also no mention of FaaS in the instructions

cshape replied:
added

cshape commented:
@JasonSowers I've updated this PR based on your good feedback (added in some more detail to the instructions and added a link to faas whitelisting). Please approve.
